### PR TITLE
Create ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57688](https://cumulocity.atlassian.net/browse/MTM-57688)
Original PR: [5797](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5797)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Availability" translation separated for JP (#5755) [GRAFT][release/cd] (#5797)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Separate "Availability" translation for Japan
```

File: content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57688
+version: 1019.7.14
+---
+"Availability" translation separated for JP (#5755) [GRAFT][release/cd] (#5797)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md:17]: The Claude AI suggested to use the following body:
```suggestion
The "Availability" text is now translated separately for Japan to improve localization. This allows the availability text to be tailored for the Japanese market.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

**Review by jdreSAG — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Separate "Availability" translation for Japan
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md:3]:
```suggestion
title: Separate translation for the term "Aavilability" for Japanese
```

File: content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57688
+version: 1019.7.14
+---
+The "Availability" text is now translated separately for Japan to improve localization. This allows the availability text to be tailored for the Japanese market.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md:17]: @janhommes What does that mean? The term "Availability"? Or what is meant by text? And what means "separately" here? Don´t get it.

Comment (jdreSAG) [reply content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md:17]: @BeateRixen according to ticket description https://cumulocity.atlassian.net/browse/MTM-57688 in japanese language they have different word for 'availability' when it's about availability of dashboards (property indicating which role should current user have to edit particular dashboard) and availability of application package (based on package app type).

Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md:17]:
```suggestion
The term "Availability" in the UI now offers different translations for Japanese depending on the context. This improvement better tailors the localization for the Japanese market.
```

**Pull request merged**